File: Faculty/TeacherManagerQtInterface/project1/student.cpp
#include "student.h"
#include <sstream>

using namespace std;
int Student::get_id()
{
	return id;
}

int Student::get_group()
{
	return group;
}

std::string Student::to_string()
{
	stringstream ss;
	ss << id << " - " << name << " (" << group << "): " << grade;
	return ss.str();

}

std::string Student::to_file()
{
	stringstream ss;
	ss << id << "," << name << "," << group << "," << grade << "\n";
	return ss.str();
}

std::string Student::get_name()
{
	return name;
}

//bool Student::operator==(const Student   s)


std::vector<int> Teacher::get_groups()
{
	return groups;	
}

StudentException::~StudentException()
{
}

const char * StudentException::what()
{
	return this->message.c_str();
}

File: Faculty/StudentControl/src/entities/student.py
class Student:
    def __init__(self, id, name, group):
        self.__id = id
        self.__name = name
        self.__group = group

    def set_name(self, value):
        self.__name = value

    def set_group(self, value):
        self.__group = value

    def get_name(self):
        return self.__name

    def get_id(self):
        return int(self.__id)

    def get_group(self):
        return int(self.__group)

    def __str__(self):
        return str(self.__id) + " - " + str(self.__name) + " (" + str(self.__group) + ") "
